priceupdater/priceupdater.go

```diff
@@ -21,180 +22,260 @@ const (
 	defaultIdleConnTimeout = 2 * time.Second
 )
 
-// UpdateMethodType defines the token price update mechanism
-type UpdateMethodType string
+// Priority defines the priority provider
+type Priority string
```

Comment:
I don't understand the reason of this type (at which point this is different from a string?)

Reply:
you are right